src/sub-components/ArticlesPanel.jsx

```diff
@@ -1,23 +1,28 @@
 import  { getArticles } from '../utils/api'
 import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
 
 const ArticlesPanel = ({topicQuery}) => {
     const [ articles, setArticles ] = useState([]);
+    const [ articleError, setArticleError ] = useState()
 
+   
     useEffect(() => {
-        getArticles(topicQuery).then((response) => {
-            setArticles(response)
-        })
-    }, [articles, topicQuery])
-
+            getArticles(topicQuery).then((response) => {
+                setArticles(response);
+            })
+        }, [articles, topicQuery])
+   
     return (
         <main>
             <ul>
                 {articles.map(article => {
                    return (
                         <li key={article.article_id}>
                             <h6>{article.topic}</h6>
+                            <Link to={`/articles/${article.article_id}`}>
                             <h3>{article.title}</h3>
+                            </Link>
                             <h4>{article.body}</h4>
```

Comment:
When we are viewing the articles in a list, it's not necessary for us to show the body of the article. If we had an article that was very long, this would take up a lot of space and would lead to a lot of unnecessary scrolling! 

We can view the body of the article upon clicking the title